backend/corpora/dataset_processing/process.py

```diff
@@ -219,12 +216,8 @@ def create_artifact(
             processing_status={processing_status_type: ConversionStatus.UPLOADED},
         )
 
-    except Exception as e:
-        update_db(
-            dataset_id,
-            processing_status={processing_status_type: ConversionStatus.FAILED},
-        )
-        raise e
+    except Exception:
+        raise Exception({processing_status_type: ConversionStatus.FAILED})
```

Comment:
Won't this raise a generic Exception and not the original exception? We wont get the same error information by calling a generic Exception.

Reply:
you can add the `processing_status` to the exception so it can be used in the exception handler in `main`

```
e.cellxgene_processing_status = {processing_status_type: ConversionStatus.FAILED}
```

This way we can still preserve the original exception. Or you can log the exception here instead and then call your own exception.